config/description.md

```diff
@@ -1,14 +1,14 @@
 
-We thank the [GISAID Initiative](https://gisaid.org) and the [GISRS Network](http://www.who.int/influenza/gisrs_laboratory/en/) for critical surveillance efforts and open data sharing. Titer data used in antigenic analyses was generated by the [Influenza Division at the US Centers for Disease Control and Prevention](https://www.cdc.gov/flu/), the [Worldwide Influenza Centre at the Francis Crick Institute](http://www.crick.ac.uk/research/worldwide-influenza-centre), the [Victorian Infectious Diseases Reference Laboratory at the Australian Peter Doherty Institute for Infection and Immunity](http://www.vidrl.org.au/) and the [Influenza Virus Research Center at the Japan National Institute of Infectious Diseases](https://www.niid.go.jp/niid/en/flu-e.html). We gratefully acknowledge the authors, originating and submitting laboratories of sequences from the GISAID EpiFlu Database on which this research is based. An attribution table is available by clicking on "Download Data" at the bottom of the page and then clicking on "Strain Metadata" in the resulting dialog box.
+We thank the [GISAID Initiative](https://gisaid.org) and the [GISRS Network](https://www.who.int/initiatives/global-influenza-surveillance-and-response-system) for critical surveillance efforts and open data sharing. Titer data used in antigenic analyses was generated by the [Influenza Division at the US Centers for Disease Control and Prevention](https://www.cdc.gov/flu/), the [Worldwide Influenza Centre at the Francis Crick Institute](http://www.crick.ac.uk/research/worldwide-influenza-centre), the [Victorian Infectious Diseases Reference Laboratory at the Australian Peter Doherty Institute for Infection and Immunity](http://www.vidrl.org.au/) and the [Influenza Virus Research Center at the Japan National Institute of Infectious Diseases](https://www.niid.go.jp/niid/en/flu-e.html). We gratefully acknowledge the authors, originating and submitting laboratories of sequences from the GISAID EpiFlu Database on which this research is based. An attribution table is available by clicking on "Download Data" at the bottom of the page and then clicking on "Strain Metadata" in the resulting dialog box.
```

Comment:
Since we're updating the description here, we should refer to the new "Acknowledgments" option that replaced the "Strain metadata" option:

```suggestion
We thank the [GISAID Initiative](https://gisaid.org) and the [GISRS Network](https://www.who.int/initiatives/global-influenza-surveillance-and-response-system) for critical surveillance efforts and open data sharing. Titer data used in antigenic analyses was generated by the [Influenza Division at the US Centers for Disease Control and Prevention](https://www.cdc.gov/flu/), the [Worldwide Influenza Centre at the Francis Crick Institute](http://www.crick.ac.uk/research/worldwide-influenza-centre), the [Victorian Infectious Diseases Reference Laboratory at the Australian Peter Doherty Institute for Infection and Immunity](http://www.vidrl.org.au/) and the [Influenza Virus Research Center at the Japan National Institute of Infectious Diseases](https://www.niid.go.jp/niid/en/flu-e.html). We gratefully acknowledge the authors, originating and submitting laboratories of sequences from the GISAID EpiFlu Database on which this research is based. An attribution table is available by clicking on "Download Data" at the bottom of the page and then clicking on "Acknowledgments" in the resulting dialog box.
```